packages/vest/dist/vest.js

```diff
@@ -1,21 +1,29 @@
-"warnings";var d="warnCount"}else a="errors",d="errorCount";c.tests[b][a]=c.tests[b][a]||[];e&&c.tests[b][a].push(e);c[d]++;c.tests[b][d]++}},setPending:function(b){g=!0;a.push(b)},addToSkipped:function(a){!c.skipped.includes(a)&&c.skipped.push(a)},markAsDone:function(a){c.canceled||(a&&(h(a),l(a.fieldName)||k(a.fieldName)),l()||k())},pending:a,output:c}};return l.register({Enforce:z.Enforce,VERSION:"1.0.9",any:F,draft:function(){var b=l.useContext();if(b)return b.result.output;k("draft hook called outside of a running suite.")},
-enforce:z,only:function(b){return E("only",b)},runWithContext:y,skip:function(b){return E("skip",b)},test:function(b){for(var a,d,e=arguments.length,g=Array(1<e?e-1:0),h=1;h<e;h++)g[h-1]=arguments[h];"string"===typeof g[0]?(a=g[0],d=g[1]):"function"===typeof g[0]&&(d=g[0]);if("function"===typeof d){a=new t(l.useContext(),b,a,d);d=a.ctx;g=a.fieldName;e=!1;if((h=l.useContext())&&h.exclusive&&(h.exclusive.skip&&h.exclusive.skip[g]||h.exclusive.only&&!h.exclusive.only[g]))d.result.addToSkipped(g);else{d.result.markTestRun(g);
-a.ctx.setCurrentTest(a);try{var k=a.testFn.apply(a)}catch(J){k=!1}a.ctx.removeCurrentTest();!1===k&&a.fail();k&&"function"===typeof k.then&&(e=!0,a.testFn=k);e&&d.result.setPending(a)}return a}},validate:function(b,a){if("string"!==typeof b)return k("Suite initialization error. Expected name to be a string.",TypeError);if("function"!==typeof a)return k("Suite initialization error. Expected tests to be a function.",TypeError);var d=I(b);y({result:d},function(){a();var b=d.pending;var g=Array.isArray(b)?
-w(b):void 0;g||(g="undefined"!==typeof Symbol&&Symbol.iterator in Object(b)?Array.from(b):void 0);if(!g)a:{if(b){if("string"===typeof b){g=w(b,void 0);break a}g=Object.prototype.toString.call(b).slice(8,-1);"Object"===g&&b.constructor&&(g=b.constructor.name);if("Map"===g||"Set"===g){g=Array.from(g);break a}if("Arguments"===g||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(g)){g=w(b,void 0);break a}}g=void 0}if(!(b=g))throw new TypeError("Invalid attempt to spread non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.");
-b.forEach(H)});return d.output},warn:function(){var b=l.useContext();b?b.currentTest?b.currentTest.warn():k("warn hook called outside of a test callback. It won't have an effect."):k("warn hook called outside of a running suite.")}})})
+'use strict';(function(v,r){"object"===typeof exports&&"undefined"!==typeof module?module.exports=r():"function"===typeof define&&define.amd?define(r):(v=v||self,v.vest=r())})(this,function(){function v(d,a){return a={exports:{}},d(a,a.exports),a.exports}function r(d){"@babel/helpers - typeof";r="function"===typeof Symbol&&"symbol"===typeof Symbol.iterator?function(a){return typeof a}:function(a){return a&&"function"===typeof Symbol&&a.constructor===Symbol&&a!==Symbol.prototype?"symbol":typeof a};
```

Comment:
Would consider getting this out and to put it in .gitignore

Reply:
This is intentional. The idea is that one can download vest as a script file to use on their site.
But maybe this really is not needed anymore in 2020  🤔